### DISCUSSION TITLE
Mend-hosted app config docs uses different values than `forkProcessing` seems to allow

### DISCUSSION THREAD

**Discussion by HonkingGoose** [renovatebot/renovate#32215] [Request Help] [answered]
### How are you running Renovate?

None

## Intro

I think I found a problem with the docs:

-  The config option docs for `forkProcessing` lists these `allowedValues` (string type!):
   - `"auto"`
   - `"enabled"`
   - `"disabled"`
- But the Mend-hosted app configuration page says you can use a Boolean `true` value, this is not a `allowedValue`?

### Questions for maintainers

- [ ] Are the instructions for the Mend-hosted app configuration still correct? Maybe the Mend backend allows a boolean `true` option?
- [ ] Should the Mend-hosted app configuration page be updated to use the string values from the `allowedValues` list from the `forkProcessing` config option?

## Sources

Quote from [Renovate docs, `forkProcessing` config option](https://docs.renovatebot.com/configuration-options/#forkprocessing):

> | Name | Value |
> | --- | --- |
> | Type | string |
> | allowedValues | auto, enabled, disabled |
> | default | `"auto"` |
> | cli | `--fork-processing` |
> | env | `RENOVATE_FORK_PROCESSING` |

But the new Mend-hosted app configuration page says that if you want to run Renovate on a fork, you should set `forkProcessing` to `true`. But `true` is a boolean value, not a string. And `true` is not in the `allowedValues` list.

Quote from [Renovate docs, Mend-hosted Apps Configuration](https://docs.renovatebot.com/mend-hosted/hosted-apps-config/#fork-processing):

> ## Fork Processing
> 
> If an Organization installs Renovate with the "All repositories" option, then `forkProcessing` will remain set to its default value `false`.
> This means forked repositories are _not_ onboarded, Renovate ignores them.
> To change this behavior, push a `renovate.json` file to the repository with `"forkProcessing": true`.
> 
> If an Organization installs Renovate with "Selected repositories", we assume the organization wants to onboard _all_ of the selected repositories, even forked repositories.
> Therefore we set `forkProcessing` to `true`.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
@justo-mend type in docs?

**Comment by jamietanna** [comment, marked as answer]
Corrected in #33712
